exec/main.c

```diff
@@ -1198,6 +1289,7 @@ int main (int argc, char **argv, char **envp)
 					"        -f     : Start application in foreground.\n"\
 					"        -p     : Do not set realtime scheduling.\n"\
 					"        -r     : Set round robin realtime scheduling (default).\n"\
+					"        -R     : Do not try move corosync to root cpu cgroup\n" \
```

Comment:
I can agree it's suboptimal and ifdef would be not that bad but I'm trying to avoid ifdef as much as possible. But I don't have strong opinion here so @chrissie-c what do you think?